models/migrations/v1_21/v271.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_21 //nolint
+
+import (
+	"xorm.io/xorm"
+)
+
+func OAuth2ApplicationAddLockedProperty(x *xorm.Engine) error {
```

Comment:
You need to reference this function on `migrations.go`.

Reply:
Oops, my first time creating a migration :laughing: Could've used the proposed migration docs from #23793